### DISCUSSION TITLE
Google colab

### DISCUSSION THREAD

**Discussion by Woisek** [vladmandic/sdnext#679] [Q&A] [answered]
Google Colab: https://colab.research.google.com/drive/126cDNwHfifCyUpCCQF9IHpEdiXRfHrLN

Any chance, to get the possibility, to use a checkpoint from google drive itself and not been limited to this few? 🙂

**Comment by vladmandic** [comment]
Can you report to Colab author?

**Reply by Woisek** [reply]
Oh ... I thought you did this ... 😋 My bad ...
Who's the author of this?

**Reply by vladmandic** [reply, marked as answer]
i've noted it in my update: <https://github.com/vladmandic/automatic/discussions/99#discussioncomment-5773255>

**Reply by Woisek** [reply]
Totally missed it, thanks ... 🤪
